src/main/java/at/grahsl/kafka/connect/mongodb/MongoDbSinkConnectorConfig.java

```diff
@@ -120,7 +123,52 @@ public MongoDbSinkConnectorConfig(Map<String, String> parsedConfig) {
     }
 
     public static ConfigDef conf() {
-        return new ConfigDef()
+        return new ConfigDef() {
+
+            <T> Validation<T> ensureValid(String name, Consumer<T> consumer) {
+                return new Validation<T>(name, consumer);
```

Comment:
IDE suggests option to remove explicit type T and replace with Validation<>

Reply:
https://github.com/941design/kafka-connect-mongodb/commit/0b94adcffd44dc43493bd7a154bd58fc3c7775e7